packages/features/eventtypes/components/CreateEventTypeDialog.tsx

```diff
@@ -166,76 +187,70 @@ export default function CreateEventTypeDialog({
         enableOverflow
         title={teamId ? t("add_new_team_event_type") : t("add_new_event_type")}
         description={t("new_event_type_to_book_description")}>
-        <Form
-          form={form}
-          handleSubmit={(values) => {
-            createMutation.mutate(values);
-          }}>
-          <div className="mt-3 space-y-6 pb-11">
-            {teamId && (
+        {!teamId ? (
+          <Form
+            form={form}
+            handleSubmit={(values) => {
+              createMutation.mutate(values);
+            }}>
+            <div className="mt-3 space-y-6 pb-11">
               <TextField
-                type="hidden"
-                labelProps={{ style: { display: "none" } }}
-                {...register("teamId", { valueAsNumber: true })}
-                value={teamId}
+                label={t("title")}
+                placeholder={t("quick_chat")}
+                data-testid="event-type-quick-chat"
+                {...register("title")}
+                onChange={(e) => {
+                  form.setValue("title", e?.target.value);
+                  if (form.formState.touchedFields["slug"] === undefined) {
+                    form.setValue("slug", slugify(e?.target.value));
+                  }
+                }}
               />
-            )}
```

Comment:
Love that this simplified, CreateEventTypeDialog as well. It had too many `teamId` conditions scattered.